frontend/dispatcher.lua

```diff
@@ -332,11 +341,30 @@ function Dispatcher:init()
     for i=1,#CreOptions do
         parseoptions(CreOptions, i)
     end
+    UIManager:broadcastEvent(Event:new("DispatcherRegisterActions"))
     Dispatcher.initialized = true
 end
 
+--[[--
+    Adds settings at runtime.
+    see helloworld plugin for an example.
+
+    @param name: the key to use in the table
+    @param value: a table per settingsList above.
```

Comment:
```suggestion
Adds settings at runtime.

See the helloworld plugin for an example.

@param name: the key to use in the table
@param value: a table per settingsList above.
```

Reply:
You can btw add it to the config, with `examples = {'examples', exclude = {'examples/slow.lua'}}`, and link to it.

Reply:
> You can btw add it to the config, with `examples = {'examples', exclude = {'examples/slow.lua'}}`, and link to it with `@see`.

?
I don't get this.
I am trying to use the docs from https://stevedonovan.github.io/ldoc/manual/doc.md.html Do you have a better reference  for me to use?
Thanks for your help with this.

Reply:
I copied that from there. ;-) You don't have to leave it as an exercise to the reader to find the hello world plugin. At its simplest by linking to it on GH, but I was saying you could experiment with that too if you want.

Reply:
Then you can link to it with `@script` or something, I haven't really used that functionality yet.

Reply:
fixed